README.md

```diff
@@ -257,7 +229,7 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
```

Comment:
Please remove this line because it's only an instruction of what you can put here